.github/workflows/avm.platform.set-avm-github-pr-labels.yml

```diff
@@ -0,0 +1,41 @@
+# Workflow for adding labels to newly created PRs
+name: .Platform - Set AVM GitHub PR labels
+
+on:
+  pull_request:
```

Comment:
Does this work for PRs open from a fork?

Reply:
I guess so

Reply:
I think this trigger only works for PRs open from a branch, but I may be wrong

Reply:
@eriqua that is a repository setting, see https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#enabling-workflows-for-forks-of-private-repositories

Reply:
Not sure I follow, the provided link is mentioning private repositories

Reply:
@eriqua you are right, my bad. Overall a great catch from your side. I changed the trigger to pull_request_target, see https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target